packages/protocol/src/interfaces/IChiefAccessHelper.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity 0.8.15;
+
+/**
+ * @title Chief helper interface.
+ * @author Fujidao Labs
+ * @notice Defines interface for {Chief} access control operations.
+ */
+
+import {IAccessControl} from "openzeppelin-contracts/contracts/access/IAccessControl.sol";
+
+interface IChiefAccessHelper is IAccessControl {
```

Comment:
Why should be called "Helper"? I guess we can call it simply "IChief"

Reply:
renamed. Only comment here is that I am not enforcing the interface over Chief contract. Only `timelock()` is needed to be known externally by our contracts.